property.go

```diff
@@ -92,19 +92,19 @@ func (client *Client) ListPropertyDefinitions(variables *PayloadVariables) (Prop
 		variables = client.InitialPageVariablesPointer()
 	}
 	if err := client.Query(&q, *variables, WithName("PropertyDefinitionList")); err != nil {
-		return PropertyDefinitionConnection{}, err
+		return &PropertyDefinitionConnection{}, err
```

Comment:
After reading through all of these - i think if we are going to switch to returning a pointer these error checks should really return nil for the object.

```suggestion
		return nil, err
```

Reply:
Yes! I forgot about this so it's an oversight on my end.

Reply:
Changed